Cargo.toml

```diff
@@ -21,13 +27,6 @@ gdk_3_14 = ["gdk-sys/gdk_3_14", "gdk_3_12"]
 
 [dependencies]
 libc = "0.1"
-
-[dependencies.gdk-sys]
-path = "gdk-sys"
-version = "0.1.0"
-
-[dependencies.glib]
-git = "https://github.com/rust-gnome/glib.git"
-
-[dependencies.glib-sys]
-git = "https://github.com/rust-gnome/glib.git"
+gdk-sys = "^0"
```

Comment:
Well, in my case I only target 0.*

2015-05-12 22:03 GMT+02:00 Gleb Kozyrev <notifications@github.com>:

> In Cargo.toml
> <https://github.com/rust-gnome/gdk/pull/36#discussion_r30174048>:
>
> > @@ -21,13 +27,6 @@ gdk_3_14 = ["gdk-sys/gdk_3_14", "gdk_3_12"]
> >
> >  [dependencies]
> >  libc = "0.1"
> > -
> > -[dependencies.gdk-sys]
> > -path = "gdk-sys"
> > -version = "0.1.0"
> > -
> > -[dependencies.glib]
> > -git = "https://github.com/rust-gnome/glib.git"
> > -
> > -[dependencies.glib-sys]
> > -git = "https://github.com/rust-gnome/glib.git"
> > +gdk-sys = "^0"
>
> That's some interesting syntax. I think people use "*" for that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-gnome/gdk/pull/36/files#r30174048>.
>



-- 
*web CV : www.guillaume-gomez.fr <http://www.guillaume-gomez.fr>*
*portable: +33 6 21 92 31 88*